LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+Copyright (c) 2003-2021 Eelco Dolstra and the Nixpkgs/NixOS contributors
+
```

Comment:
this repo contains package recipes mostly developed/created by BBP/Adrien. Do we need NixOS license here? 🤔

Reply:
Unless Adrien is a Nix dev, I don't really see either Eelco's name nor anyone unfamiliar in the contributors list above.

Reply:
I think this is the same concern we have w/ the spack packages, ie: [# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other](https://github.com/BlueBrain/spack/blob/e3a9532c86e075ecd50020ed5ca95776edf68b7b/var/spack/repos/builtin/packages/neurodamus-base/package.py#L2)

I vaguely remember a discussion on that, do you remember?

Reply:
yeah, its _similar_ but in Spack those copyright headers are created by Spack commands and checked under CI for consistency and so on. So more work was needed in case of Spack compared to this specific case where top level license file seems to be just copied.

Anyway, with the recent work from @matz-e of re-organising BBP specific packages in separate Spack repo, we can check if we can avoid LLNL copyrights.

Reply:
> Anyway, with the recent work from @matz-e of re-organising BBP specific packages in separate Spack repo, we can check if we can avoid LLNL copyrights.

That's not done yet. Groundwork has been laid, but this will only be finalized with the new deployment.

Reply:
@pramodk > yeah, its similar [...]  checked under CI for consistency and so on [...]

I don't think it's much different here; other than the boilerplate copied rather than be generated by spack

I would consider removing this repository, as I'm not sure it needs to be public, or that anyone uses it (and if they are, it could be mirrored internally.)